authors/apps/notifications/models.py

```diff
@@ -0,0 +1,33 @@
+from django.db import models
+from ..authentication.models import User
+
+
+class Article(models.Model):
```

Comment:
Did you sync with the team working on the articles module on this?

Reply:
Yes I did, I consulted them on how they had implemented it before adding this. This model is just for running tests, once they push the Article model I will delete it and import from them.